src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1829,9 +1829,13 @@ public abstract partial class Enum : System.ValueType, System.IComparable, Syste
         public bool HasFlag(System.Enum flag) { throw null; }
         public static bool IsDefined(System.Type enumType, object value) { throw null; }
         public static bool IsDefined<TEnum>(TEnum value) where TEnum : struct, System.Enum { throw null; }
+        public static object Parse(System.Type enumType, System.ReadOnlySpan<char> value) { throw null; }
```

Comment:
Did you manually edited this file or did you use GenerateReferenceSource to edit it? I only ask since we want future adds/changes to API usually use the target to edit and we would want to prevent showing diffs that aren't part of that future PR.

Reply:
Manually.

Reply:
@hrrrrustic could you please use the flag described here ot make the edit: https://github.com/dotnet/runtime/blob/4d784693ebc5f91c7eede32170046355ef3969b2/docs/coding-guidelines/updating-ref-source.md

Reply:
@danmosemsft Done